multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -144,3 +176,23 @@ write_files:
           name: kubelet-context
         current-context: kubelet-context
 
+  - path: /etc/kubernetes/cni/net.d/10-calico.conf
```

Comment:
It doesn't look like this is being created on the master nodes. Is it not necessary for some reason?